documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -149,13 +244,10 @@ EOF
 <1> Specify the name of the `Plan` CR.
 <2> Specify whether the migration is warm or cold. If you specify a warm migration without specifying a value for the `cutover` parameter in the `Migration` CR manifest, only the precopy stage will run.
 <3> You can create multiple network mappings for source and destination networks.
-<4> You can use either the `id` _or_ the `name` parameter to specify the source network.
-<5> _Managed object reference_ of the source network.
-<6> You can create multiple storage mappings for source data stores and destination storage classes.
-<7> You can use either the `id` _or_ the `name` parameter to specify the source data store.
-<8> _Managed object reference_ of the source data store.
-<9>  You can use either the `id` _or_ the `name` parameter to specify the source VM.
-<10> _Managed object reference_ of the source VM.
+<4> Specify the name of the `NetworkMap` CR.
+<5> Specify the name of the `StorageMap` CR.
+<6> You can use either the `id` _or_ the `name` parameter to specify the source VM.
+<7> Specify the managed object reference of the source VM.
 
```

Comment:
Please correct the numbers in the example - should be (6) and (7) instead of (9) and (10)